vignettes/covidHubUtils-overview.Rmd

```diff
@@ -118,6 +175,41 @@ p + scale_x_date(name=NULL, date_breaks = "1 month", date_labels = "%b") +
 Additionally, `plot_forecasts()` can handle plotting multiple models or locations or forecast dates at the same time as the following examples show.
 
 
+## as_of versioning
+
+```{r message=FALSE}
+# Load forecasts that were submitted in a time window from zoltar
+```
+
+## Hub Location
+
+```{r message=FALSE}
+# Load forecasts that were submitted in a time window from zoltar
+inc_case_targets <- paste(1:4, "wk ahead inc case")
+forecasts_ECDC <- load_latest_forecasts(models=c("ILM-EKF"),
```

Comment:
set verbose to FALSE to prevent query messages from zoltar

Reply:
This is done